cmd/query/app/grpc_handler_test.go

```diff
@@ -332,6 +349,17 @@ func TestArchiveTraceNotFoundGRPC(t *testing.T) {
 	})
 }
 
+func TestArchiveTraceEmptyTraceFailureGRPC(t *testing.T) {
+	withServerAndClient(t, func(server *grpcServer, client *grpcClient) {
+
+		_, err := client.ArchiveTrace(context.Background(), &api_v2.ArchiveTraceRequest{
+			TraceID: model.TraceID{},
+		})
+		assert.Error(t, err)
```

Comment:
Same here, you can assert the concrete error with Equal.

Reply:
Updated